packages/keystone/fetch.spec.js

```diff
@@ -0,0 +1,85 @@
+jest.mock('node-fetch', () => jest.fn())
```

Comment:
I think it is better to test this agains express app

Check out initTestExpressApp

Reply:
In spec files much easier to mock node_module